inc/lift/Header.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <string>
+#include <string_view>
+
+namespace lift {
+
+class Header {
+public:
+    /**
+     * Creates an owned header.
+     * @param name The name of the header.
+     * @param value The value of the header.
+     */
+    explicit Header(
+        std::string_view name,
+        std::string_view value);
+
+    /**
+     * @return The entire header, e.g. "Connection: Keep-Alive"
+     */
+    [[nodiscard]] auto HeaderFull() const -> const std::string& { return m_header; }
+    [[nodiscard]] auto HeaderFull() -> std::string& {return m_header; }
```

Comment:
this could be friended to Executor and made private, kind of terrible to expose it as mutable ref with the views looking into it

Reply:
fixed